src/components/displayBooks.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import AddBook from './form';
+import SingleBook from './singleBook';
+
+const DisplayBooks = () => (
+  <div className="book-list">
+    <button type="submit">Remove</button>
+    <SingleBook />
+    <AddBook />
+  </div>
+);
```

Comment:
- [ ] Kindly be noticed that you should render a list of books using the `SingleBook` component.
- [ ] Also you should remove the remove button as it should be in the `SingleBook` component.

```
const DisplayBooks = () => 
const books = [];
(
  <div className="book-list">
   books.map((book) => {
       return <SingleBook title={book.title} author={book.author} />
    }
    <AddBook />
  </div>
);
```